templates/Admin/base.html.twig

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
daca ai un base deja, n-as recomanda in mod special sa faci altul specific pentru admin. Poti sa il folosesti pe acelasi.